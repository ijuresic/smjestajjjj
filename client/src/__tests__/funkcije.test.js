require('text-encoding').TextEncoder = require('util').TextEncoder;

const React = require('react');
const { shallow, configure } = require('enzyme');
const Adapter = require('@wojtekmaj/enzyme-adapter-react-17');

// Konfiguracija adaptera
configure({ adapter: new Adapter() });

describe('Funkcije', () => {
  it('renders correctly', () => {
    const Funkcije = React.createElement('div', { className: 'funkcije' }, 'Uspješan test!');
    const wrapper = shallow(Funkcije);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders the correct content', () => {
    const Funkcije = React.createElement('div', { className: 'funkcije' }, 'Uspješan test!');
    const wrapper = shallow(Funkcije);
    expect(wrapper.text()).toContain('Uspješan test!');
  });
});
