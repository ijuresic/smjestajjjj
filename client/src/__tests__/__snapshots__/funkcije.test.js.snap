// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Funkcije renders correctly 1`] = `ShallowWrapper {}`;
